<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiPreferencesDialog</class>
 <widget class="QDialog" name="GuiPreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>831</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Score Settings</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="5" column="1">
         <widget class="QComboBox" name="followStopPointCombo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="showNoteNamesCheck">
          <property name="text">
           <string>Show Note Names</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="timingMarkersCheck">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Timing Markers</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="showColoredNotesCheck">
          <property name="toolTip">
           <string>Show color coded notes on the score</string>
          </property>
          <property name="text">
           <string>Color Coded Notes</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="courtesyAccidentalsCheck">
          <property name="text">
           <string>Courtesy Accidentals</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Follow stop point:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="followThroughErrorsCheck">
          <property name="text">
           <string>Follow Through Errors</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>250</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>500</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>723</width>
           <height>590</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="10" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QSpinBox" name="noteDimRedSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="noteDimGreenSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="noteDimBlueSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="15" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QSpinBox" name="barMarkerRedSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="barMarkerGreenSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="barMarkerBlueSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Stave Color:</string>
            </property>
           </widget>
          </item>
          <item row="18" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QSpinBox" name="noteNameRedSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="noteNameGreenSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="noteNameBlueSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="17" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QSpinBox" name="pianoBadRedSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="pianoBadGreenSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="pianoBadBlueSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="20" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Playing Zone Middle Line Color:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QSpinBox" name="staveDimRedSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="staveDimGreenSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="staveDimBlueSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="14" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QSpinBox" name="bgRedSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="bgGreenSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="bgBlueSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="13" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Play Stopped Color:</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Play Bad Color:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QSpinBox" name="menuSelectedRedSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="menuSelectedGreenSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="menuSelectedBlueSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="15" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Bar Marker Color:</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QSpinBox" name="noteRedSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="noteGreenSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="noteBlueSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Stave Color Dim:</string>
            </property>
           </widget>
          </item>
          <item row="21" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Playing Zone End Lines Color:</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QSpinBox" name="staveRedSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="staveGreenSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="staveBlueSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="19" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Playing Zone Area Color:</string>
            </property>
           </widget>
          </item>
          <item row="17" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Piano Bad Color:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QSpinBox" name="menuRedSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="menuGreenSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="menuBlueSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Menu Selected Color:</string>
            </property>
           </widget>
          </item>
          <item row="13" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QSpinBox" name="playStoppedRedSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="playStoppedGreenSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="playStoppedBlueSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Played Good Color:</string>
            </property>
           </widget>
          </item>
          <item row="16" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QSpinBox" name="beatMarkerRedSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="beatMarkerGreenSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="beatMarkerBlueSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="14" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Background Color:</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Note Color:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Menu Color:</string>
            </property>
           </widget>
          </item>
          <item row="19" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QSpinBox" name="playZoneBgRedSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="playZoneBgGreenSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="playZoneBgBlueSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="11" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QSpinBox" name="playGoodRedSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="playGoodGreenSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="playGoodBlueSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Note Color Dim:</string>
            </property>
           </widget>
          </item>
          <item row="12" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QSpinBox" name="playBadRedSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="playBadGreenSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="playBadBlueSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="21" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QSpinBox" name="playZoneEndLineRedSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="playZoneEndLineGreenSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="playZoneEndLineBlueSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="20" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QSpinBox" name="playZoneMiddleRedSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="playZoneMiddleGreenSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="playZoneMiddleBlueSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="16" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Beat Marker Color:</string>
            </property>
           </widget>
          </item>
          <item row="18" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Note Name Color:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Music Course</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QCheckBox" name="showTutorPagesCheck">
          <property name="text">
           <string>Show Tutor Help Pages</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Language</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Language:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="languageCombo">
          <property name="maximum" stdset="0">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GuiPreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GuiPreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
